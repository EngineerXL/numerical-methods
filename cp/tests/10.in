580017028804315133110679
6685551192141751928939422
