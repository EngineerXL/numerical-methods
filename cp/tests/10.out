3877733538385205905562462618450790716616212287538
